h2o-docs/src/product/data-science/upliftdrf.rst

```diff
@@ -0,0 +1,470 @@
+
+Defining a Uplift DRF Model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  `model_id <algo-params/model_id.html>`__: (Optional) Specify a custom name for the model to use as
+   a reference. By default, H2O automatically generates a destination
+   key.
+
+-  `training_frame <algo-params/training_frame.html>`__: (Required) Specify the dataset used to build the
+   model. **NOTE**: In Flow, if you click the **Build a model** button from the
+   ``Parse`` cell, the training frame is entered automatically.
+
+-  `validation_frame <algo-params/validation_frame.html>`__: (Optional) Specify the dataset used to evaluate
+   the accuracy of the model.
+
+-  `y <algo-params/y.html>`__: (Required) Specify the column to use as the dependent variable. The data can be only categorical (only binomial classification is currently supported).
+
+-  `x <algo-params/x.html>`__: Specify a vector containing the names or indices of the predictor variables to use when building the model. If ``x`` is missing, then all columns except ``y`` are used.
+
+-  `score_each_iteration <algo-params/score_each_iteration.html>`__: (Optional) Enable this option to score
+   during each iteration of the model training. This option is defaults to false (not enabled).
+
+-  `score_tree_interval <algo-params/score_tree_interval.html>`__: Score the model after every so many trees.
+   Disabled if set to 0 (default).
+
+-  `ignored_columns <algo-params/ignored_columns.html>`__: (Optional, Python and Flow only) Specify the column or columns to be excluded from the model. In Flow, click the checkbox next to a column
+   name to add it to the list of columns excluded from the model. To add
+   all columns, click the **All** button. To remove a column from the
+   list of ignored columns, click the X next to the column name. To
+   remove all columns from the list of ignored columns, click the
+   **None** button. To search for a specific column, type the column
+   name in the **Search** field above the column list. To only show
+   columns with a specific percentage of missing values, specify the
+   percentage in the **Only show columns with more than 0% missing
+   values** field. To change the selections for the hidden columns, use
+   the **Select Visible** or **Deselect Visible** buttons.
+
+-  `ignore_const_cols <algo-params/ignore_const_cols.html>`__: Specify whether to ignore constant
+   training columns, since no information can be gained from them. This option is defaults to true (enabled).
+
+-  `balance_classes <algo-params/balance_classes.html>`__: Oversample the minority classes to balance the class distribution. This option is defaults to false (not enabled), and can increase the data frame size. This option is only applicable for classification.
```

Comment:
Does it make sense for the Uplift? How it handles treatment col?

Reply:
Good point. Should be disabled for now.